_episodes/01-introduction.md

```diff
@@ -161,7 +170,7 @@ If you do not include a start index, the slice automatically starts at `list_nam
 > {: .solution}
 {: .challenge}
 
-## For loops
+## Repeating an operation many times: for loops
 Often, you will want to do something to every element of a list.  The structure
 to do this is called a `for` loop.  The general structure of a `for` loop is
```

Comment:
For the section where the importance of indentation is discussed. A few points to make:
 - Indentation changes when a `:` is used (this must always be present at the end of the `for` loop.
 - The Jupyter notebook will fix your indentation automatically when a `:` is present on the previous line.